roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -15,6 +15,9 @@
       disable_push_dockerhub: "{{ openshift_disable_push_dockerhub | default(None) }}"
       hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(None) }}"
       hosted_registry_network: "{{ openshift_docker_hosted_registry_network | default(None) }}"
+  - role: common
+    local_facts:
+      openshift_deployment_type: "{{ deployment_type }}"
```

Comment:
Why would we persist this as openshift.common.openshift_deployment_type? shouldn't it just be openshift.common.deployment_type?  

Also, this should be pushed down into openshift_facts, which is a dependency of openshift_docker_facts.